01-basics/20-weather/WeatherApp.js

```diff
@@ -4,44 +4,80 @@ import { getWeatherData, WeatherConditionIcons } from './weather.service.ts'
 export default defineComponent({
   name: 'WeatherApp',
 
+  setup() {
+    const data = getWeatherData()
+
+    function tempConverter(temp) {
+      const convertedTemp = (temp - 273.15).toFixed(1)
+      return convertedTemp
+    }
+
+    function pressureConverter(pressure) {
+      const convertedPressure = Math.round(pressure * 0.75)
+      return convertedPressure
+    }
+
+    function pickIcon(id) {
+      const icon = WeatherConditionIcons[id]
+      return icon
+    }
+
+    function isNightTheme({ dt, sunrise, sunset }) {
+      let currentTime = new Date()
+      currentTime.setHours(dt.split(':')[0], dt.split(':')[1])
+      let sunriseTime = new Date()
+      sunriseTime.setHours(sunrise.split(':')[0], sunrise.split(':')[1])
+      let sunsetTime = new Date()
+      sunsetTime.setHours(sunset.split(':')[0], sunset.split(':')[1])
+      return currentTime < sunriseTime || sunsetTime < currentTime
+    }
```

Comment:
When time is `HH:MM` string in a 24h format, it can be compared as a string without the need of converting to a number.